docs/monorepo.md

```diff
@@ -266,13 +266,14 @@ Most projects in the monorepo should have a mirror repository holding a built ve
    1. The repo's description should begin with `[READ ONLY]` and end with `This repository is a mirror, for issue tracking and development head to: https://github.com/automattic/jetpack`.
    2. The default branch should be `trunk`, matching the monorepo.
    3. In the repo's settings, turn off wikis, issues, projects, and so on.
-   4. Make sure that [matticbot](https://github.com/matticbot) can push to the repo.
+   4. Make sure that [matticbot](https://github.com/matticbot) can push to the repo. You would do this here: `https://github.com/Automattic/example-reposiroty-name/settings/branches` - creating a new branch protection rule where only Matticbot (and whoever needs access to push, for example Ground Control) can push to that repository.
    5. Make sure that Actions are enabled. The build process copies workflows from `.github/files/mirror-.github` into the mirror to do useful things like automatically close PRs with a reference back to the monorepo.
    6. Create any secrets needed (e.g. for Autotagger or Npmjs-Autopublisher). See PCYsg-xsv-p2#mirror-repo-secrets for details.
-2. If your project requires building, configure `.scripts.build-production` in your project's `composer.json` to run the necessary commands.
-3. If there are any files included in the monorepo that should not be included in the mirror, use `.gitattributes` to tag them with "production-exclude".
-4. If there are any built files in `.gitignore` that should be included in the mirror, use `.gitattributes` to tag them with "production-include".
-5. Set `.extra.mirror-repo` in your project's `composer.json` to the name of the repo.
+2. For a package you also need to go to packagist.org and create the package there. This requires pushing a first commit with a valid `composer.json` to the repository.
```

Comment:
Should we add extra details on the Packagist submission here? Ideally, one will want to add `automattic`, `kraft`, and `dereksmart` as maintainers after adding the package to Packagist.

Reply:
I think that would be a great idea. I don't know what this involves in detail - I assume we could link to this Packagist page: https://packagist.org/about#how-to-submit-packages, then in terms of the first commit something along these lines: p1658238330799389/1658237219.448209-slack-C01U2KGS2PQ (if I understand correctly, which I don't think I am). Unless we can link to another example of what a valid `composer.json` entry should look like - perhaps one from another package. Then add those maintainers too.

Reply:
> in terms of the first commit something along these lines: p1658238330799389/1658237219.448209-slack-C01U2KGS2PQ (if I understand correctly, which I don't think I am). Unless we can link to another example of what a valid composer.json entry should look like - perhaps one from another package.

I personally tend to copy the new package's `composer.json` from the PR that introduces it. As I review the PR, I have all the info I need there. You do not really need to copy anything else into the mirror repo, since everything will get added once the PR is merged.

Reply:
Added these details in now, thanks.